chapters/annotations.tex

```diff
@@ -1117,13 +1117,16 @@ \section{Annotations for the Graphical User Interface}\label{annotations-for-the
 
 This section describes the annotations that are used to define properties of the graphical user interface.
 
-\begin{lstlisting}[language=grammar]
-  preferred-view-annotation:
-     annotation "(" preferredView "=" ("info" | "diagram" | "text") ")"
+\begin{lstlisting}[language=modelica]
+annotation(preferredView = $layer$)
 \end{lstlisting}
 
 The \fmtannotationindex{preferredView} annotation defines the default view when selecting the class.
-\lstinline!info! means info layer, i.e., the documentation of the class, \lstinline!diagram! means diagram layer and \lstinline!text! means the Modelica text layer.
+The $layer$ is a \lstinline!String! literal where \lstinline!"info"! means info layer, i.e., the documentation of the class, \lstinline!"diagram"! means diagram layer and \lstinline!"text"! means the Modelica text layer.
+
+\begin{nonnormative}
+Tools may also support these values without quotes, e.g., \lstinline!info!, as a deprecated feature.
+\end{nonnormative}
```

Comment:
Deprecated means normatively deprecated.  We shouldn't introduce a new habit of having non-normatively deprecated features.  By the way, what would being non-normatively deprecated even mean – more or less a standard language feature compared to other deprecated features?

```suggestion
A deprecated variant of the annotation is to use identifiers instead of \lstinline!String! literals, for example, \lstinline!preferredView = info!.
```

Reply:
Normatively making it a deprecated feature would mean that all tools must support it, until it is removed.

Requiring all tools to support `preferredView=info` does not make sense.

Reply:
Remove.  (Not sure why GitHub doesn't let me make a suggestion.)